spinalcordtoolbox/scripts/sct_deepseg.py

```diff
@@ -188,6 +188,16 @@ def get_parser(subparser_to_return=None):
         help="Contrast of the input. Specifies the contrast order of input images (e.g. `-c t1 t2`)",
         choices=('t1', 't2', 't2star'),
         metavar=Metavar.str)
+    parser_dict['totalspineseg'].add_argument(
+        "-step1-only",
+        type=int,
+        help="If set to '1', only step one will be performed.\n"
+             "- Step 1: <>\n"
+             "- Step 2: <>\n"
```

Comment:
Whoops! This was meant to be a placeholder that I filled in once I dug into the outputs. Fixing this now, too. :)